mesh/mesh_initializer.py

```diff
@@ -41,9 +41,53 @@ class Mesh():
     mesh.mesh_center:
         - Stores the coordinates of the position vector of the center of the smallest box that could fit the mesh.
         - Dimension: (3 spatial coordinates)
+
+    mesh.mesh_orientation:
+        - store the 3 orthonormal basis vectors that define the mesh orientation.
+        - Dimension: (3 spatial coordinates, 3 orthonormal basis vectors)
+        - Initial value: [[1,0,0],[0,1,0],[0,0,1]]
+
+    Methods:
+    --------
+
+    mesh.mesh_update():
+    Parameters: None
+        - This method updates/refreshes the mesh attributes in pyelastica geometry.
+        - By default this method is called at initialization and after every method that might change the mesh attributes.
+
+    mesh.visualize():
+    Parameters: None
+        - This method visualizes the mesh using pyvista.
+
+    mesh.translate():
+    Parameters: {numpy.ndarray-(3 spatial coordinates)}
+    ex : mesh.translate(np.array([1,1,1]))
+        - This method translates the mesh by a given vector.
+        - By default, the mesh's center is at the origin;
+          by calling this method, the mesh's center is translated to the given vector.
+
+    mesh.scale():
+    Parameters: {numpy.ndarray-(3 spatial constants)}
+    ex : mesh.scale(np.array([1,1,1]))
+        - This method scales the mesh by a given factor in respective axes.
+
+    mesh.rotate():
+    Parameters: {rotation_axis: unit vector[numpy.ndarray-(3 spatial coordinates)], angle: in degrees[float]}
+    ex : mesh.rotate(np.array([1,0,0]), 90)
+        - This method rotates the mesh by a given angle about a given axis.
     """
+
     def __init__(self, filepath: str) -> None:
         self.mesh = pv.read(filepath)
+        self.orientation_cube = pv.Cube()
+        self.mesh_update()
+
+    def mesh_update(self) -> None:
+        """
+        This method updates/refreshes the mesh attributes in pyelastica geometry.
+        This needs to be performed at the first initialization as well as
+        after every method that might change the mesh attributes.
+        """
         self.mesh_center = self.mesh.center
         self.pyvista_face_normals = self.mesh.face_normals
         self.pyvista_faces = self.mesh.faces
```

Comment:
why do we need a variable called `pyvista_*`

Reply:
These variables are required to calculate the desired attributes; though separate variable assigning is not necessary.
Should I add the arguments in object-orientated mod/without creating variables?

Reply:
Yes I think object oriented way is better

Reply:
Okay, I will implement this.